pyshield/stencils/shallow_convection/README.md

```diff
@@ -0,0 +1,12 @@
+# samfshalconv
+
+Python implementation of the scale-aware mass flux scheme shallow convection in SHiELD built using the NDSL domain-specific language middleware in Python.
```

Comment:
```suggestion
Performance portable implementation of the scale-aware mass flux scheme shallow convection in SHiELD built using the NDSL domain-specific language middleware in Python.
```

Small nitpick: I think we should be careful about claiming Python at large in our comms. NDSL is the magic here, python is a vessel